services/github/github-release.service.js

```diff
@@ -1,32 +1,85 @@
 'use strict'
 
+const Joi = require('@hapi/joi')
 const { addv } = require('../text-formatters')
-const { fetchLatestRelease } = require('./github-common-fetch')
+const { version: versionColor } = require('../color-formatters')
+const { latest } = require('../version')
 const { GithubAuthV3Service } = require('./github-auth-service')
-const { documentation } = require('./github-helpers')
+const {
+  fetchLatestRelease,
+  releaseInfoSchema,
+} = require('./github-common-fetch')
+const { documentation, errorMessagesFor } = require('./github-helpers')
+const { NotFound, redirector } = require('..')
 
-module.exports = class GithubRelease extends GithubAuthV3Service {
+const queryParamSchema = Joi.object({
+  include_prereleases: Joi.equal(''),
+  sort: Joi.string()
+    .valid('date', 'semver')
```

Comment:
another wishlist item from me not directly related to this PR, but it'd be cool if the modal window for the query params could have a dropdown for cases like these like we do for route params